docs/architecture/TECHNICAL_DATA_MODEL.md

```
@@ -425,6 +425,82 @@ File-system based permissions:
+```
+.pm/locks/
+├── issues/
+│   └── issue-001.lock
+├── epics/
+│   └── epic-001.lock
+└── projects/
+    └── project.lock
+```
+
+Lock file format:
+```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)